pkg/urbit/daemon/main.c

```diff
@@ -991,6 +993,84 @@ _cw_intr_win(c3_c* han_c)
 }
 #endif
 
+c3_c 
+*_cw_eval_get_input(FILE* fp, size_t size)
+{
+//The size is extended by the input with the value of the provisional
+  c3_c *str;
+  c3_i ch;
+  size_t len = 0;
+
+  str = realloc(NULL, sizeof(*str)*size);//size is start size
+  
+  if(!str)
+    return str;
+  
+  while(EOF!=(ch=fgetc(fp))){
+    str[len++]=ch;
+    if(len==size){
```

Comment:
This looks like it will only reallocate once. Seems like it should be `!(len % size)`, or that size should be increased after the reallocation.

Reply:
This is wrong, I misread the following line.